src/main/java/org/terasology/web/servlet/HttpAPIServlet.java

```diff
@@ -26,37 +26,45 @@
 import javax.ws.rs.GET;
 import javax.ws.rs.POST;
 import javax.ws.rs.Path;
+import javax.ws.rs.PathParam;
 import javax.ws.rs.Produces;
 import javax.ws.rs.core.Context;
 import javax.ws.rs.core.MediaType;
 import javax.ws.rs.core.Response;
 import java.util.HashMap;
+import java.util.List;
 import java.util.Map;
 import java.util.UUID;
 
 @Path("http")
```

Comment:
I know that this wasn't touched in this PR, but I'd rather use something like "api" for the root path.